.github/workflows/code-health.yml

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
I've excluded some actions present in AtlasCLI code health as I didn't see their relevance to a Kubernetes plugin.
 These actions are:
- code coverage
- library owners
- action lint
- mocks
- shell check
- check templates
- verify image


Would be interested to hear if you disagree with any of the exclusions

Reply:
I think that the following should be added, but we can do in a follow-up PR to avoid big PRs


- code coverage: we added this as a scope goal, if it's too much to add this we can cover in a separate ticket
- action lint: I think this will lint our GH actions? so I think it's good to keep it if that's what it does
- mocks: not now, but we should add it as we'll need to create mocks for K8s, and this ensures all mocks are created. can be a separate ticket too
- shell check: Worth keeping as we are already adding .sh files, this will lint them

Reply:
Awesome, added action lint, mocks and shell check as they were simple additions. Code coverage will require setting up github apps to enable sending coverage messages in the PR and might be better suited to a separate PR.